PIL/Image.py

```diff
@@ -1092,12 +1154,14 @@ def draft(self, mode, size):
         pass
```

Comment:
@bjd2385 Documentation is pulled from Image.Image, but the implementations of the file formats inherit from Image -> ImageFile -> JpegImagePlugin.